packages/spectral/src/types/IntegrationDefinition.ts

```diff
@@ -161,6 +155,49 @@ export interface Flow<TTriggerPayload extends TriggerPayload = TriggerPayload> {
   onExecution: FlowOnExecution<TTriggerPayload>;
 }
 
+export type StandardTriggerType = "standard";
+
+/** A standard flow with a webhook or scheduled trigger (non-polling). */
+interface StandardFlow<TTriggerPayload extends TriggerPayload = TriggerPayload>
+  extends FlowBase<TTriggerPayload> {
+  triggerType?: StandardTriggerType;
+  /** Schedule configuration that defines the frequency with which this flow will be automatically executed. */
+  schedule?: (ValueExpression<string> | ConfigVarExpression) & {
+    timezone?: string;
+  };
+  /** Specifies the trigger function for this flow, which returns a payload and optional HTTP response. */
+  onTrigger?:
+    | TriggerReference
+    | TriggerPerformFunction<Inputs, ConfigVars, false, TriggerResult<false, TTriggerPayload>>;
+}
+
+export type PollingTriggerType = "polling";
+
+/** A polling flow that runs on a schedule and has access to polling context (getState/setState). */
+interface PollingFlow<TTriggerPayload extends TriggerPayload = TriggerPayload>
+  extends FlowBase<TTriggerPayload> {
+  /**
+   * Type of trigger for this flow. A "polling" trigger runs on a schedule
+   * and can use context.polling.* functions. Requires schedule to be set.
+   */
+  triggerType: PollingTriggerType;
+  /** Schedule configuration that defines the frequency with which this flow will be automatically executed. Required for polling triggers. */
+  schedule: (ValueExpression<string> | ConfigVarExpression) & {
```

Comment:
The distinct type ensures schedule is required for `triggerType: "polling"` it will also be enforced via a runtime check.